testprojects/src/python/python_targets/python_distribution/superhello/hello_package/hello.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import super_greet
+
+def hello():
+  print(super_greet.super_greet())
```

Comment:
Remove? I assume this was for debugging.

Reply:
Nope. That returns a string which needs to be printed for verification in stdout.